basesite/models.py

```diff
@@ -73,6 +73,11 @@ class Genre(models.Model):
     name = models.CharField("Имя", max_length=100)
     url = models.SlugField(max_length=160, unique=True)
 
+    # def save(self, *args, **kwargs):
+    #     super().save(*args, **kwargs)
+    #     self.url = slugify(self.name, allow_unicode=True) + '-' + str(self.id)
+    #     super().save(update_fields=['url'])
```

Comment:
Why did you commented this out?

Reply:
because, will been bug with genres, and i off many def's. for understending situation